src/dev/src/app/app.routing.ts

```diff
@@ -27,10 +27,10 @@ export const APP_ROUTES: Routes = [
     loadChildren: () => import('./checkboxes/checkboxes.demo.module').then(m => m.CheckboxesDemoModule),
   },
   { path: 'color', loadChildren: () => import('./color/color.demo.module').then(m => m.ColorDemoModule) },
-  {
-    path: 'combobox',
-    loadChildren: () => import('./combobox/combobox.demo.module').then(m => m.ComboboxDemoModule),
-  },
+  // {
```

Comment:
See component comment regarding demo for combobox

Reply:
Sorry for the force push needed to split the commits and restart the CI to make sure the build was working properly